syscheck/dirs.go

```diff
@@ -70,3 +70,44 @@ func checkSnapMountDir() error {
 
 	return nil
 }
+
+var (
+	// distributions known to use /usr/lib/snapd/
+	defaulLibExectDirDistros = []string{
+		"ubuntu",
+		"ubuntu-core",
+		"ubuntucoreinitramfs",
+		"debian",
+		"opensuse-leap",
```

Comment:
Huh, did we really deviate in leap vs tumbleweed?

Reply:
it'd likely that some old version did not set opensuse-leap, but 15.6 does (and other versions are EOL)

Reply:
to be clear, we need to find out what's the situation for leap 16, but I don't think the changed libexecdir